chapters/en/slides/module2/module2_29.Rmd

```diff
@@ -85,22 +86,26 @@ Notes: Script here
 
 ---
 
-Let's go through the steps that are needed to make this plot. 
+Now let's say we are interested in plotting the sugar content in cereals from each manufacturer. We do this using a `bar` plot which we are quite familiar with to do this, we will be using the `mfr` column and the `sugars` column from the `cereal` dataframe we have been using. 
+
+Here, `mfr` is a categorical column (remember this explanation from module 1?). This is because there is no specific order in which cereal type must come before another. For example, it is not the case that "Kelloggs" must come before “Quaker”. Because of this, the `mfr` column is categorized as a Nominal data type. The `sugars` column in contrast, is categorized as quantitative column. It is a quantitative column since it is a quantitative measurement that takes on any positive integer value.
```

Comment:
Instead of capitalizing Nominal put it in ittalic and/or bold.

Reply:
Can we rephrase "It is a quantitative column since it is a quantitative ...." thats sounds off.